index.js

```diff
@@ -400,7 +400,8 @@ export const Classes = {
 		recommended: 'recommended',
 		startDate: 'start-date',
 		timeLimitEnforced: 'time-limit-enforced',
-		notimelimit: 'notimelimit'
+		notimelimit: 'notimelimit',
+		retaken: 'retaken'
```

Comment:
since the `retaken` class is applied to the specific question response entity, we should put it under `questions` instead of `quizzes`

looks good otherwise!